example/package.json

```diff
@@ -27,20 +27,20 @@
   },
   "dependencies": {
     "@cumulus/api": "^1.10.1",
-    "@cumulus/common": "^1.10.0",
+    "@cumulus/common": "^1.10.1",
     "@cumulus/deployment": "^1.10.1",
-    "@cumulus/discover-granules": "^1.10.0",
-    "@cumulus/discover-pdrs": "^1.10.0",
-    "@cumulus/hello-world": "^1.10.0",
+    "@cumulus/discover-granules": "^1.10.1",
+    "@cumulus/discover-pdrs": "^1.10.1",
+    "@cumulus/hello-world": "^1.10.1",
     "@cumulus/integration-tests": "^1.10.1",
-    "@cumulus/move-granules": "^1.10.0",
-    "@cumulus/parse-pdr": "^1.10.0",
-    "@cumulus/pdr-status-check": "^1.10.0",
-    "@cumulus/post-to-cmr": "^1.10.0",
-    "@cumulus/queue-granules": "^1.10.0",
-    "@cumulus/queue-pdrs": "^1.10.0",
-    "@cumulus/sf-sns-report": "^1.10.0",
-    "@cumulus/sync-granule": "^1.10.0",
+    "@cumulus/move-granules": "^1.10.1",
+    "@cumulus/parse-pdr": "^1.10.1",
+    "@cumulus/pdr-status-check": "^1.10.1",
+    "@cumulus/post-to-cmr": "^1.10.1",
+    "@cumulus/queue-granules": "^1.10.1",
+    "@cumulus/queue-pdrs": "^1.10.1",
+    "@cumulus/sf-sns-report": "^1.10.1",
+    "@cumulus/sync-granule": "^1.10.1",
     "@cumulus/test-data": "^1.10.0",
     "@cumulus/test-processing": "^1.10.1",
     "@cumulus/wait-for-deploy": "^1.10.1",
```

Comment:
This doesn't actually have an npm package, so setting it up outside a current dev environment fails - will this be added as part of the release process?

Reply:
you're right about this. Originally I was thinking that we shouldn't publish it, but we have to because when we run the tests on npm when we do the next release it will break if we don't publish this. I think what I need to do is set it to publish in the next release but somehow not advertise on npm.